packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -250,6 +251,24 @@ export const SlackSingleChannelNotifier: NotificationIntegrationHelper<SlackNoti
   async integrationUpdated() {
     // Slack sends a system message on its own
     return 'success'
+  },
+
+  async standupResponseSubmitted(meeting, team, user, response) {
+    const options = {
+      searchParams: {
+        utm_source: 'slack standup submission',
+        utm_medium: 'product',
+        utm_campaign: 'notifications',
+        responseId: response.id
+      }
+    }
+    const responseUrl = makeAppURL(appOrigin, `meet/${meeting.id}/responses`, options)
```

Comment:
+1 if i go to /responses it doesn't open the right drawer for me. it's the same behavior if i leave off the /responses. 
is that expected?

Reply:
Are you including a `responseId` query param? It gets set as part of the `searchParams` in the `options` var above, so `/responses` alone just goes to the `responses` phase of the meeting, which is the only phase of standups.

If you connect your local dev to our #test-slack-bot channel (or some other test channel) and submit a response, you should see a message with a link that opens the correct drawer.

Reply:
ahh that must be it. Is it weird that there's no query param when using our UI? e.g. i click "reply" & i goto /responses without a param so i can't share the link. definitely off topic, just sent me down a wormhole

Reply:
Yeaaaaa that's something we've talked about on the squad in the past. IIRC there's something non-trivial about keeping the URL in-sync, but I don't quite remember what. One option we discussed was an explicit "share link" button in the discussion drawer to copy the link with the param without the param being in the browser URL. At the end of the day, it's just something we've never prioritized.